lstchain/reco/r0_to_dl1.py

```diff
@@ -322,7 +322,6 @@ def r0_to_dl1(input_filename = get_dataset_path('gamma_test_large.simtel.gz'),
                 tel.prefix = ''  # don't really need one
                 # remove the first part of the tel_name which is the type 'LST', 'MST' or 'SST'
                 tel_name = str(event.inst.subarray.tel[telescope_id])[4:]
-                tel_name = tel_name.replace('-003', '')
```

Comment:
sorry @vuillaut I'm a bit confused now with the other discussion in cta-observatory/ctapipe_io_lst#27, what would be the `tel_name` that is arriving at this point when using the `LSTCam-004` geometry?

Reply:
`tel_name` is always `LSTCam` whatever the geometry, the version is already stripped